debian/changelog

```diff
@@ -1,3 +1,13 @@
+runc (1.1.0-0ubuntu1~20.04.2) focal; urgency=medium
+
+  * Fix byte order while parsing cpuset range to bits (LP: #1993221)
+    - upstream, cgroups: cpuset: fix byte order while parsing cpuset range
+      to bits #3611
+  * Add patch
+    - d/p/fix_cpuset_range_byte_order.patch
```

Comment:
Taking a better look at the changelog entry I think we can simplify to something like this:

```
  * d/p/fix_cpuset_range_byte_order.patch: fix byte order while parsing cpuset
    range to bits (LP: #1993221)
```

Reply:
Thanks for your comments. I have modified the changelog content.